components/eamxx/src/physics/mam/eamxx_mam_aci_functions.hpp

```diff
@@ -406,6 +406,10 @@ void call_function_dropmixnuc(
               }
             });
         team.team_barrier();
+        // HACK: dropmixnuc() requires the parameter enable_aero_vertical_mix,
+        //       so we define it here until we have a better idea of where it
+        //       might come from
+        const bool enable_aero_vertical_mix = true;
```

Comment:
Is this ALWAYS going to be true? The fact that `dropmixnuc` has it as an input arg makes me wonder if there are cases we foresee needing this to be false? If it's ALWAYS going to be true, this is fine. If we foresee it being false sometimes, then you could maybe add it as an input to `call_function_dropmixnuc`, and have the process interface read it from input file (defaulting to `true` if not found).